test/editor/Extensions/TextmateTokenizerTests.re

```diff
@@ -221,7 +221,6 @@ describe("Textmate Service", ({test, _}) => {
 
         switch (tokenizeResult, colorMap^) {
         | (Some(v), Some(cm)) =>
-          Console.error(v.colors);
```

Comment:
Removed this whilst I was running the tests since it was printing whilst the test run. Can undo if its needed to get extra information out during CI?

Reply:
Oh, no, accidentally left this in - thanks for removing it!